src/main/kotlin/calculator/StringAdditionCalculator.kt

```diff
@@ -0,0 +1,41 @@
+package calculator
+
+class StringAdditionCalculator {
+    fun calculate(expression: String?): Int {
+        if (expression.isNullOrBlank()) {
+            return 0
+        }
+
+        if (expression.contains(CUSTOM_DELIMITER_REGEX)) {
+            val matchResult = CUSTOM_DELIMITER_REGEX.find(expression)!!
+            val (_, customDelimiter, numberAdditionExpression) = matchResult.groupValues
```

Comment:
destructuring 활용 👍 

-----
Regex의 destructured를 쓰면 0번째를 제외할 수 있습니다.
```suggestion
            val (customDelimiter, numberAdditionExpression) = matchResult.destructured
```